### ISSUE TITLE
[BE] Text inserted in the frontend is tranferred to AI

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#31]
Expansion to #30 

## Acceptance criteria
- [ ] Text that was inserted via the frontend is tranferred to the AI integrated into the backend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
